pkg/user/handler_test.go

```diff
@@ -0,0 +1,238 @@
+package user
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/dhis2-sre/im-manager/pkg/config"
+	"github.com/dhis2-sre/im-manager/pkg/model"
+	"github.com/dhis2-sre/im-manager/pkg/token"
+	"github.com/gin-gonic/gin"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+)
+
+func TestHandler_RefreshToken_Cookie(t *testing.T) {
```

Comment:
Why did you not add a test to user_integration_test.go? Mock based tests don't give me much confidence it actually works which for something involving users and auth is key 😅